apps/fluent-tester/src/TestComponents/CheckboxExperimental/CheckboxTest.tsx

```diff
@@ -34,10 +22,14 @@ const BasicCheckbox: React.FunctionComponent = () => {
       <Checkbox label="Checked checkbox (uncontrolled)" onChange={onChangeUncontrolled} defaultChecked accessibilityLabel="Hello there" />
       <Checkbox label="Disabled checkbox" disabled />
       <Checkbox label="Disabled checked checkbox" defaultChecked disabled />
-      <Checkbox label="Checkbox will display a tooltip" tooltip="This is a tooltip" />
-      <Checkbox label="A circular checkbox" shape="circular" />
-      <Checkbox label="A checkbox with label placed before" labelPosition="before" />
-      <Checkbox label="A required checkbox" required />
+      {Platform.OS !== 'android' && (
+        <>
+          <Checkbox label="Checkbox will display a tooltip" tooltip="This is a tooltip" />
+          <Checkbox label="A circular checkbox" shape="circular" />
+          <Checkbox label="A checkbox with label placed before" labelPosition="before" />
+          <Checkbox label="A required checkbox" required />
+        </>
+      )}
```

Comment:
Maybe label this section "Desktop specific properties"?. It can then be it's own test section.

Reply:
Changes completed and approved , hence resolving.